confgenerator/confgenerator_test.go

```diff
@@ -69,6 +69,21 @@ var (
 					Platform:        "linux_platform",
 					PlatformVersion: "linux_platform_version",
 				},
+				HasNvidiaGpu: false,
```

Comment:
false is the zero value, so you can just omit it for the rest of the the platforms

Reply:
Done!